core/modules/modules.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Provides helper functions for handling user and system modules.
+ *
+ * Written By:
+ * 		Matthew Knox
+ *
+ * License:
+ *		MIT License. All code unless otherwise specified is
+ *		Copyright (c) Matthew Knox and Contributors 2015.
+ */
+
+var fs              = require('fs'),
+    path            = require('path'),
+    loaders         = [require.once('./kassyModule.js')],
+    files           = require.once('./../files.js'),
+    config          = require('./../config.js'),
+    modulesDir      = 'modules',
+    descriptor      = 'kassy.json',
+    conflict        = 1;
+
+exports.listModules = function (disabled) {
+    var modules = {};
+    for (var i = 0; i < loaders.length; i++) {
+        var m = loaders[i].listModules(disabled);
+        for (var key in m) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://www.codacy.com/app/theprojectshop/Kassy/file/2529918277/issues/source?bid=3282987&fileBranchId=3282998#l25)